FWCore/MessageService/src/ELoutput.cc

```diff
@@ -632,7 +633,7 @@ namespace edm {
     }
 
     void ELoutput::changeFile(const std::string& filename) {
-      os.reset(new std::ofstream(filename.c_str(), std::ios /*_base*/ ::app), close_and_delete());
+      os = std::make_shared<std::ofstream>(filename.c_str(), std::ios /*_base*/ ::app, close_and_delete());
```

Comment:
@makortel , as `std::make_shared` does not allow a custom deleter. so should I revert this change and add `// NOLINT` ?

Reply:
> as `std::make_shared` does not allow a custom deleter. so should I revert this change and add `// NOLINT` ?

Yes, this change should be reverted.

Reply:
So clang-tidy made a change that it then fails to process? :(